### PR TITLE
BUG-835374: Updating expired default keystore and CA certificate 

### PR DESCRIPTION
**Pull request by arvasrikanth**
Updating expired default keystore and CA certificate

### CODE REVIEW

File: charts/pega/README.md [hunk 1]
```diff
@@ -1165,7 +1165,7 @@ Parameter   | Description   | Default value
 `service.tls.traefik.insecureSkipVerify` | Set to `true` to skip verifying the certificate; do this in cases where you do not need a valid root/CA certificate but want to encrypt load balancer traffic. Leave the setting to `false` to both verify the certificate and encrypt load balancer traffic. | `false`
 
 ##### Important Points to note
-- By default, Pega provides a self-signed keystore and a custom root/CA certificate in Helm chart version `2.2.0`. To use the default keystore and CA certificate, leave the parameters service.tls.keystore, service.tls.keystorepassword and service.tls.cacertificate empty.
+- By default, Pega provides a self-signed keystore and a custom root/CA certificate in Helm chart version `2.2.0`. To use the default keystore and CA certificate, leave the parameters service.tls.keystore, service.tls.keystorepassword and service.tls.cacertificate empty.Expiry date of default keystore and CA certificate is 25/12/2025.
```
Comment (taz-pega-work) [inline charts/pega/README.md:1168]: can you make some slight improvements for readabilty?
service.tls.cacertificate empty.Expiry date
to
service.tls.cacertificate empty.*** The expiration*** date

**Review by taz-pega-work — APPROVED** [review]
please fix the typo and using of expiry but otherwise looks good!

File: charts/pega/README.md (hunk 1 above)
Comment (arvasrikanth) [reply charts/pega/README.md:1168]: @taz-pega-work They are different statements . We are shipping default keystore and CA certificate which has some expiry date which I mentioned.

**Review by kingakowalska1 — CHANGES_REQUESTED** [review]
Hi, perhaps it would be easier to just say:

"The default keystore and CA certificate expire on 25/12/2025."

This way we don't need to decide between "expiry date" and "expiration date".

**Comment by arvasrikanth** [comment]
> Hi, perhaps it would be easier to just say:
> 
> "The default keystore and CA certificate expire on 25/12/2025."
> 
> This way we don't need to decide between "expiry date" and "expiration date".

Updated the readme as suggested

**Review by kingakowalska1 — APPROVED** [review]
Great, thanks!

**Pull request merged**